src/admm/tron_kernel.jl

```diff
@@ -62,9 +65,9 @@ Driver to run TRON on GPU. This should be called from a kernel.
 
         if task == 0 || task == 1
             if use_polar
-                f = eval_f_polar_kernel(n, shift, scale, x, param, YffR, YffI, YftR, YftI, YttR, YttI, YtfR, YtfI)
+                f = eval_f_polar_kernel(n, shift, scale, x, param, YffR, YffI, YftR, YftI, YttR, YttI, YtfR, YtfI, I, J)
```

Comment:
I see we need to pass explicitly `I` and `J`. I would maybe add a comment to explain why we are doing this (I guess it's related to a KA limitation?)